eval.py

```diff
@@ -1464,23 +1466,23 @@ def main():
         print('Loading model...', end='')
 
         if args.engine == Engine.DEEPSPARSE:
-            net = DeepSparseWrapper(filepath=args.trained_model, cfg=cfg,
+            net = DeepSparseWrapper(filepath=trained_model_weights, cfg=cfg,
                                     num_cores=args.num_cores,
                                     batch_size=args.batch_size
                                     )
         elif args.engine == Engine.ORT:
-            net = ORTWrapper(filepath=args.trained_model, cfg=cfg,
+            net = ORTWrapper(filepath=trained_model_weights, cfg=cfg,
                              batch_size=args.batch_size)
         else:
             net = Yolact()
-            net.load_checkpoint(args.trained_model)
+            net.load_checkpoint(trained_model_weights)
             net.eval()
         print(' Done.')
 
         if args.cuda:
             net = net.cuda()
 
-        evaluate(net, dataset)
+        evaluate(net, dataset, device)
```

Comment:
`device` can be undefined at this point. Adding `device = 'cpu'` after L1439 should fix it.